arkouda/client.py

```diff
@@ -438,7 +646,16 @@ def _send_string_message(
 
     logger.debug(f"sending message {message}")
 
-    socket.send_string(json.dumps(message.asdict()))
+    loop = get_event_loop()
+
+    def asyncEligible(cmd: str) -> bool:
+        return cmd not in {'delete', 'connect', 'getconfig'}
```

Comment:
Moving forward, I am assuming we will have more commands if not all supported. Would we need to list every command here or would this be removed if we are supporting all commands?

Reply:
I might be wrong but I think this is saying every command except `delete`, `connect`, and `getconfig` is async eligible (i.e. will return True)

Reply:
@Ethan-DeBandi99 ah, sorry, as @pierce314159 this checks to see if a method can be run in async mode. The reasons for the three are as follows:

1. delete--since delete invokes __del__ on pdarray, this cannot be run async b/c __del__ is also invoked on Arkouda shell exit() and fails due to the resulting asyncio failing due to modules being deleted
2. connect--this is because Python client connect method is itself async-enabled
3. getconfig--this is invoked within the async Python client connect method, which, again, is itself async-enabled